From: Jan Engelhardt <jengelh@inai.de>
Date: 2020-08-13 11:01:08.470708019 +0200

Do not fail the build if munit available (only) via /usr.

---
 test/meson.build |    2 --
 1 file changed, 2 deletions(-)

Index: libei-0.5/test/meson.build
===================================================================
--- libei-0.5.orig/test/meson.build
+++ libei-0.5/test/meson.build
@@ -2,8 +2,6 @@ if not get_option('tests')
     subdir_done()
 endif
 
-subproject('munit', default_options: 'werror=false')
-
 munit = dependency('munit', fallback: ['munit', 'munit_dep'])
 
 lib_unittest = static_library('unittest',
